cfn-resources/serverless-instance/cmd/resource/resource.go

```diff
@@ -3,6 +3,7 @@ package resource
 import (
 	"context"
 	"fmt"
+	"github.com/openlyinc/pointy"
```

Comment:
Why do we need pointy? the same thing is available with aws lib.

Reply:
wich one is in the aws lib?